api/tests/routes/native/openapi_test.py

```diff
@@ -1559,9 +1559,10 @@ def test_public_api(client):
                         "ARTS_LOISIRS_CREATIFS",
                         "CARTES_JEUNES",
                         "CD_VINYLE_MUSIQUE_EN_LIGNE",
+                        "CINEMA",
                         "CONCERTS_FESTIVALS",
                         "EVENEMENTS_EN_LIGNE",
-                        "FILMS_SERIES_CINEMA",
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** 🧨 c'est un breaking change pour l'app

On devrait laisser les 3 cohabiter un temps. J'ai au moins 2 endroits de l'app où ça va casser. 

[Exemple dans le code de l'app](https://github.com/pass-culture/pass-culture-app-native/blob/de920d25ed3f46c74fe415d16a01c7d2d1268526/src/features/offer/components/OfferBody/OfferBody.tsx#L36)
[Autre exemple](https://github.com/pass-culture/pass-culture-app-native/blob/de920d25ed3f46c74fe415d16a01c7d2d1268526/src/features/search/enums.ts#L124)

Il y a aussi des fixtures à update